### DISCUSSION TITLE
Can't query for Regional Categories

### DISCUSSION THREAD

**Discussion by nilo-galleon** [wp-graphql/wp-graphql#3210] [Q&A] [answered]
### What problem does this address?

Hello!

We are building a headless wordpress and we are trying to query the regional categories, I have searched the documentation but can't find the necessary query to do so.

When I query the general categories they all appear as normal, but not the regional ones.

Do we have to create a custom query for this?

### What is your proposed solution?

Create a standardized query for regional categories just like we have with normal categories.

**Comment by justlevine** [comment, marked as answer]
What may I ask is a "regional category"?

Assuming it's a custom term, you'd need to expose it to WPGraphQL before you can query for it. Here's a [doc link](https://www.[wpgraphql.com/docs/custom-taxonomies](https://www.wpgraphql.com/docs/custom-taxonomies) if you're registering them in PHP, but if you're using something like ACF or Custom Post Types UI, there's settings in those plugins/extensions to expose specific types to the GraphQL schema.

Converting this to a discussion, since we use the issue tracker specifically for tracking development work. You can also get support from other community members in the [official WPGraphQL discord](https://discord.gg/AGVBqqyaUY).

**Reply by nilo-galleon** [reply]
Hello! Just to finish up the conversation, I'm not a Wordpress Developer, I'm actually a DevOps engineer that needed to deploy this GraphQL plugin.
The regional stuff I mentioned was actually a custom taxonomy, sorry for raising this unecessary issue, I was unaware of the terms.

Thanks for the plugin, great work from all!
